### ISSUE TITLE
wgan-gp

### ISSUE THREAD

**Issue by ymmshi** [eriklindernoren/Keras-GAN#48]
the paper said 'No critic batch normalization',but u add batch normalization every layer,why?

**Comment by Ortix92** [comment]
I wondered the same thing

**Comment by zhongtao93** [comment]
I think the writer just give us a code example, so he copy the critic network simply from wgan.py.

**Comment by Rabia-Raja** [comment]
Why there is no accuracy value in WGAN and WGAN-GP code? print function is only printing d-loss and g-loss. can someone explain me? @ymmshi @ortix  @zhongtao93
